_docs/platforms/azure-aks/index.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: guide
+title: StorageOS Docs - Azure
+anchor: platforms
+module: platforms/azure-aks
+---
+
+# AKS
+
+This section of documentation covers the use of the managed Kubernetes Azure
+service [AKS](https://azure.microsoft.com/en-gb/services/kubernetes-service/).
+For information on the installation of StorageOS with vanilla Kubernetes in Azure
+VMs, please refer to the [Kubernetes standard installation]({%link
+_docs/platforms/kubernetes/install/index.md %}) procedure.
+
+AKS deployment of Kubernetes uses Ubuntu by default with an optimized kernel.
+All versions of Ubuntu with a kernel version later than`4.15.0-1029-azure`
+meets the StorageOS prerequisites.
+
+## Kubernetes with StorageOS
+
+Kubernetes and StorageOS communicate with each other to perform actions such as
+creation, deletion or mounting of volumes. StorageOS implements communication 
+using CSI. By using CSI, Kubernetes and StorageOS communicate over a Unix domain
+socket, handled by the Kubelet in the Host.
+
+## CSI (Container Storage Interface) Note
+
+CSI is the future standard that enables storage drivers to release on their own
```

Comment:
Not sure I like the use of the word 'future' here, since we're recommending it now for production installations with AKS, and indeed it's the only option. How about 'burgeoning standard' instead?